lib/fathead/python_coverage/README.md

```diff
@@ -0,0 +1,28 @@
+# Python Coverage Fathead
+Pulls data from Coverage API Documentation at https://coverage.readthedocs.io/en/latest/api.html
+
+## Dependencies
+
+Requires Python 3. Created/Tested using Python 3.5.2
+### includes requirements.txt
+
+Dependencies are:
+* requests
+* bs4
+
+## Running
+
+* Install requirements:
+```
+$ pip3.5 install -r requirements.txt
```

Comment:
Most people are running 3.4. No need to be specific about the version number from the pip command.